src/vm/gcinfodecoder.cpp

```diff
@@ -1169,7 +1169,7 @@ void GcSlotDecoder::DecodeSlotTable(BitStreamReader& reader)
         m_SlotArray[0].Slot.RegisterNumber = regNum;
         m_SlotArray[0].Flags = flags;
 
-        UINT32 loopEnd = min(m_NumRegisters, MAX_PREDECODED_SLOTS);
+        UINT32 loopEnd = _min(m_NumRegisters, MAX_PREDECODED_SLOTS);
```

Comment:
It is one of those building with PAL_STDCPP_COMPAT enabled problems.  SOS and anything that SOS includes (src/gcdump) builds with this enabled so both the linux and pal include files can be used together with various restrictions.  And min/max -> _min/_max was one of them.

Someday we might be able to fix it, but at time it seemed very hard to impossible.